docs/external-generated/packages/powerpack-azure-cache/documents/overview.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Overview of the Nx powerpack-azure-cache Plugin
+description: The powerpack-azure-cache Nx plugin enables you to use Azure Storage to host your remote cache instead of Nx Cloud
+---
+
+The `@nx/powerpack-azure-cache` plugin enables you to use [Azure Storage](https://aws.amazon.com/azure) instead of Nx Cloud to host your remote cache.
```

Comment:
```suggestion
The `@nx/powerpack-azure-cache` plugin enables you to use [Azure Storage](https://azure.microsoft.com/en-us/products/storage/blobs) instead of Nx Cloud to host your remote cache.
```